sdk/storage/perf-tests/storage-blob-track-1/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@azure/perf-test-storage-blob-track-1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@azure/core-http": "^1.2.0",
```

Comment:
do we need to depend directly on `core-http`?

Reply:
Good catch, it was only for the uuid, can be removed.

Reply:
Done.